rules/aip0158/response_repeated_first_field.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0158
+
+import (
+	"fmt"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var responseRepeatedFirstField = &lint.MessageRule{
+	Name:   lint.NewRuleName(158, "response-repeated-first-field"),
+	OnlyIf: isPaginatedResponseMessage,
+	LintMessage: func(m *desc.MessageDescriptor) []lint.Problem {
+		// Rule check: for all messages that end in Response and contain a next_page_token field.
+		// Throw a linter warning if, the first field in the message (by both position and field number) is not repeated
+		nextPageToken := m.FindFieldByName("next_page_token")
```

Comment:
See comment from above. Maybe there's a good reason to require a check for the `next_page_token` field?

Reply:
@lukesneeringer should know better. summon!

Reply:
I was planning on checking for `string next_page_token` _instead of_ checking for ListFooResponse or SearchFooResponse as message names.

I would be fine with a disjunction. I would also be fine with just `string next_page_token`, since this is the pagination AIP after all.

Slight preference for just `string next_page_token`.

Reply:
Removed the next_page_token check. It is covered by `isPaginatedResponseMessage` function now.
https://github.com/googleapis/api-linter/blob/master/rules/aip0158/aip0158.go#L44-L47